controller/place.js

```diff
@@ -16,7 +16,7 @@ function setup( backend ){
     var query = req.clean.ids.map( function(id) {
       return {
         _index: 'pelias',
-        _type: id.type,
+        type: id.types,
```

Comment:
ah good idea. also, does anyone know what happens differently in elasticsearch for _type vs type? maybe @missinglink does